### ISSUE TITLE
[TOTK] Recall animation is incorrect

### ISSUE THREAD

**Issue by xorpix** [yuzu-emu/yuzu#10493]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Early Access 3621

### Description of Issue

The problem is with the way Recall is being initiated. Normally when you activate Recall, you see the spot with moving edges which starts to expand until it fills the entire screen. In my case it became the expanding ideal circle. The issue occurs using Vulkan, but OpenGL seems to be fine. 
![image](https://github.com/yuzu-emu/yuzu/assets/22274805/e666d052-efb6-41e2-bf1e-a3575834dcc9)

### Expected Behavior

The spot with dynamic edges, not an ideal circle.

### Reproduction Steps

I'm not sure about this. It used to be normal. If i remember right, it started to happen after i applied recall to Zonai's time bomb.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11592852/yuzu_log.txt)

### System Configuration

CPU: AMD Ryzen 7 5800H
GPU/Driver: RTX3070 Laptop
RAM: 32GB DDR4
OS: Windows 11 22621.1778

**Comment by goldenx86** [comment]
Try again with all mods disabled.

**Comment by xorpix** [comment]
> Try again with all mods disabled.

Didn't change anything. Still ideal circle.

**Comment by Squall-Leonhart** [comment]
set gpu accuracy to high or extreme

**Comment by xorpix** [comment]
> set gpu accuracy to high or extreme

Already emulating at High, but will try extreme.

**Comment by xorpix** [comment]
> set gpu accuracy to high or extreme

Didn't help either. Ideal circle at Extreme accuracy as well.

**Comment by Squall-Leonhart** [comment]
remove shader cache, 
remove disable fsr shader mod.
set resolution scale back to 1x.
set anisotropic filtering back to default.

**Comment by xorpix** [comment]
> remove shader cache, remove disable fsr shader mod. set resolution scale back to 1x. set anisotropic filtering back to default.

I've found the culprit. I did everything as You said and it came back to normal (tested multiple times on many things, including time bomb). After that i switched scaling to 2x and it became broken again. Here's the latest log:
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11593098/yuzu_log.txt)

**Comment by kidoman** [comment]
Yep resolution scaling seems to be breaking it, independent of mods.

**Comment by 0757myt** [comment]
After testing version to version, the earliest build which this issue happens is 3618, and the latest version which doesn't have this issue is 3617. Something in 3618 must've broken this

**Comment by Squall-Leonhart** [comment]
Buffer cache changes, shouldn't have altered how the internal scaler works though.

**Comment by Squall-Leonhart** [comment]
This appears to be a pipeline state issue,  if you clear the vulkan pipelines, the issue does not happen, it continues to not happen each start that you clear the pipelines.

Let the game load and use a pipeline state from the previous run, the issue manifests.

**Comment by scorpion81** [comment]
I have also been testing this for a while and imho this commit (which fixes the gloom texture)
https://github.com/yuzu-emu/yuzu/pull/10433/commits/01c4568786631515ea13ec8cf991d534272e87b7
seems to be the culprit. I reverted its changes locally and the recall animation works, but gloom textures are broken again.
For reference, EA 3607 doesnt have this, while since EA 3608 this issue starts happening.

**Comment by Sincasios** [comment]
I play with vulkan, 1x high (RTX 3060) and the animation is OK.

**Comment by subwolf22** [comment]
i think it could be caused by the resolution scaling setting my to1x fixed it for me

**Comment by scorpion81** [comment]
> I play with vulkan, 1x high (RTX 3060) and the animation is OK.

Does it work also after deleting the vulkan pipeline cache ? I had it "kinda" working too, but deleting the cache, and letting the shaders rebuild seems to have it broken for me again.

**Comment by Sincasios** [comment]
I said that it was OK for me, but yesterday I had the animation broken, so it should have something that makes the behavior random.

**Comment by corvik** [comment]
Still broken with latest EA also Mainline

**Comment by Squall-Leonhart** [comment]
its not broken at 1x.

**Comment by xorpix** [comment]
For me it's also not dependent on pipeline cache, because at some point cleaning pipeline cache stops solving this problem. The only thing which works is turning off upscaler.

**Comment by corvik** [comment]
Its broken also with default settings and withouts mods. Maybe its problem with AMD gpus?   Happen to me on Steam deck os 3.4.8 and 3.5.    Playing for 3 weeks, everytime with latest version and its broken acros all yuzu versions.

**Comment by xorpix** [comment]
Not only for AMD. I have Nvidia GPU.

**Comment by yyk725** [comment]
Will this bug be fixed? I use every method you said but the animation is still inccorrect

**Comment by xhaiben** [comment]
will this bug be fixed?
